.github/workflows/ci.yml

```diff
@@ -31,3 +31,14 @@ jobs:
 
   run-reuse-workflow:
     uses: opalmedapps/.github/.github/workflows/reuse.yaml@main
+
+  check-thirdparty-notice:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.2.2
+        with:
+          persist-credentials: false
```

Comment:
Only realizing this now that the action does not do this. But it does make sense.

I wonder if we should make `setup-uv` optional. For example, those pipelines that already install uv then won't have to do it a second time.

Reply:
Yes, since actions are reused in a job it does make sense to leave the checkout to the calling job, not the action itself.

> Isn't each job independent though? So we'd still have to direct it to install uv at the beginning of the job.

Each job yes. I was thinking of the case where this is added to an existing job. For example, I would add it to the `lint` job in `admin`. See here: https://github.com/opalmedapps/opal-admin/actions/runs/15716239162/job/44286557111?pr=1617

The good thing is that it finds the already installed `uv` so maybe it is fine as is.

Reply:
Ah okay, I see what you mean then. That sounds good, and if we run into any issues in projects with `uv`, we can revisit this.